quixstreams/app.py

```diff
@@ -158,8 +172,61 @@ def __init__(
             `StreamingDataFrame.process()`.
         :param on_producer_error: triggered when RowProducer fails to serialize
             or to produce a message to Kafka.
+
+        ***Quix Platform Parameters***
+
+        :param quix_config_builder: instance of `QuixKafkaConfigsBuilder` to be used
+            instead of the default one.
+            > NOTE: It is recommended to just use quix_sdk_token instead.
         """
         configure_logging(loglevel=loglevel)
+
+        # We can't use os.getenv as defaults (and have testing work nicely)
+        # since it evaluates getenv when the function is defined.
+        # In general this is just a most robust approach.
+        broker_address = broker_address or os.getenv("Quix__Broker__Address")
+        quix_sdk_token = quix_sdk_token or os.getenv("Quix__Sdk__Token")
+        consumer_group = consumer_group or os.getenv(
+            "Quix__Consumer_Group", "quixstreams-default"
+        )
+
+        if quix_config_builder and quix_sdk_token:
+            raise warnings.warn(
+                "'quix_config_builder' is not necessary when an SDK token is defined; "
+                "we recommend letting the Application generate it automatically"
+            )
+
+        if quix_sdk_token and not quix_config_builder:
+            quix_config_builder = QuixKafkaConfigsBuilder(quix_sdk_token=quix_sdk_token)
+
+        if broker_address and quix_config_builder:
+            raise ValueError("Cannot provide both broker address and Quix SDK Token")
+        elif not (broker_address or quix_config_builder):
+            raise ValueError("Either broker address or Quix SDK Token must be provided")
+        elif quix_config_builder:
+            # SDK Token or QuixKafkaConfigsBuilder were provided
+            logger.info(
+                f"QUIX APPLICATION DETECTED; it will connect to Quix Platform brokers"
```

Comment:
1. Quix Platform -> Quix Cloud

2. Let's expand the "detected" here so it's clear how exactly it was detected that it's a Quix app.  
E.g. "Quix SDK Token detected; the application will connect to the broker in Quix Cloud"